### PR TITLE
[MIBM-565][SM]: Fix for issue 1 - /review-goods redirection

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

File: app/uk/gov/hmrc/merchandiseinbaggage/controllers/Navigator.scala [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  updatedGoodsEntries: GoodsEntries,
+  declarationJourney: DeclarationJourney,
+  overThresholdCheck: Boolean,
+  callBack: DeclarationJourney => Future[DeclarationJourney])
+    extends NavigationRequestsAsync
+
+class Navigator {
+
+  def nextPage(request: NavigationRequests): Call = request match {
+    case RequestByPass(url)                                    => Navigator.nextPage(url)
```
Comment (andy1138) [inline app/uk/gov/hmrc/merchandiseinbaggage/controllers/Navigator.scala:51]: What happens if the `url` is not one of the items in the list `nextPage` ?

Comment (sureshhmrc) [reply app/uk/gov/hmrc/merchandiseinbaggage/controllers/Navigator.scala:51]: i just copy pasted the logic from PF, so if any issues, maybe we can tackle separately... 
if we have a working journey, it unblocks QA and then we can tidyup...

Comment (sureshhmrc) [reply app/uk/gov/hmrc/merchandiseinbaggage/controllers/Navigator.scala:51]: as of now, if the url is not in the list it throws a match error

Comment (andy1138) [reply app/uk/gov/hmrc/merchandiseinbaggage/controllers/Navigator.scala:51]: That's what I guessed. maybe something like `.getOrElse( errorpage )` wants to be added (and for the other requests)

**Pull request merged**